shell/platform/windows/uwp_flutter_window.h

```diff
@@ -0,0 +1,176 @@
+
+  void SetEventHandlers();
+
+  void OnDpiChanged(
+      winrt::Windows::Graphics::Display::DisplayInformation const& args,
+      winrt::Windows::Foundation::IInspectable const&);
+
+  void OnPointerPressed(winrt::Windows::Foundation::IInspectable const&,
+                        winrt::Windows::UI::Core::PointerEventArgs const& args);
+
+  void OnPointerReleased(
+      winrt::Windows::Foundation::IInspectable const&,
+      winrt::Windows::UI::Core::PointerEventArgs const& args);
+
+  void OnBoundsChanged(
+      winrt::Windows::UI::ViewManagement::ApplicationView const& appView,
+      winrt::Windows::Foundation::IInspectable const&);
+
+  void OnPointerMoved(winrt::Windows::Foundation::IInspectable const&,
+                      winrt::Windows::UI::Core::PointerEventArgs const& args);
+
+  void OnPointerWheelChanged(
+      winrt::Windows::Foundation::IInspectable const&,
+      winrt::Windows::UI::Core::PointerEventArgs const& args);
+
+  void OnKeyUp(winrt::Windows::Foundation::IInspectable const&,
+               winrt::Windows::UI::Core::KeyEventArgs const& args);
+
+  void OnKeyDown(winrt::Windows::Foundation::IInspectable const&,
+                 winrt::Windows::UI::Core::KeyEventArgs const& args);
+
+  void OnCharacterReceived(
+      winrt::Windows::Foundation::IInspectable const&,
+      winrt::Windows::UI::Core::CharacterReceivedEventArgs const& args);
+
+  void OnGamePadLeftStickMoved(double x, double y);
+
+  void OnGamePadRightStickMoved(double x, double y);
+
+  void OnGamePadButtonPressed(
+      winrt::Windows::Gaming::Input::GamepadButtons buttons);
+
+  void OnGamePadButtonReleased(
+      winrt::Windows::Gaming::Input::GamepadButtons buttons);
+
+  void OnGamePadControllersChanged();
+
+  winrt::Windows::UI::Composition::Visual CreateCursorVisual();
+
+  void StartGamepadCursorThread();
+
+  void SetupGamePad();
+
+  void SetupXboxSpecific();
+
+  double GetPosX(winrt::Windows::UI::Core::PointerEventArgs const& args);
+  double GetPosY(winrt::Windows::UI::Core::PointerEventArgs const& args);
+
+  winrt::Windows::Foundation::IAsyncAction worker_loop_{nullptr};
+
+  // The backing CoreWindow. nullptr if not set.
+  winrt::Windows::UI::Core::CoreWindow window_{nullptr};
+
+  // A pointer to a FlutterWindowsView that can be used to update engine
+  // windowing and input state.
+  WindowBindingHandlerDelegate* binding_handler_delegate_;
+
+  winrt::Windows::UI::Composition::Compositor compositor_{nullptr};
+
+  winrt::Windows::UI::Composition::CompositionTarget target_{nullptr};
+
+  winrt::Windows::UI::Composition::ContainerVisual visual_tree_root_{nullptr};
+
+  winrt::Windows::UI::Composition::Visual cursor_visual_{nullptr};
+
+  winrt::Windows::UI::Composition::SpriteVisual sprite_visual_{nullptr};
+
+  std::unique_ptr<WinRTGamePad> game_pad_{nullptr};
+
+  bool game_controller_thread_running_;
+
+  bool running_on_xbox_;
+
+  float xbox_overscan_x_offset_;
+  float xbox_overscan_y_offset_;
```

Comment:
These can all be initialized here instead of in the constructor.

Reply:
Fixed in next commit